### PR TITLE
email with map prompt

### PR DESCRIPTION
**Pull request by codemutatio**
**What does this PR do**

1. Send a mail to the emergency response team when an accident report is submitted.

2. Integrated google static map api to generate a map view of the accident location.

### CODE REVIEW

File: server/controllers/reportControllers.js [hunk 1]
```diff
@@ -1,28 +1,47 @@
...
 const db = require('./promise').ReportDb;
 
+const messageBody = (coords, link) => `<b>Emergency </b> <br></br><br></br>
+    <img src="https://maps.googleapis.com/maps/api/staticmap?
+    center=${coords[0]},${coords[1]}
+    &zoom=20&size=500x500
+    &markers=color:red%7Clabel:S%7C${coords[0]},${coords[1]}
+    &key=${MAP_API_KEY}"> <br></br><br></br> 
+    ${link}`;
+
```
Comment (Way2nnadi) [inline server/controllers/reportControllers.js:26]: something like this
```
const messageBody = (coords, link) => {
    const [coord1, coord2] = coord2;
    
    return `<b>Emergency </b> <br></br><br></br>
        <img src="https://maps.googleapis.com/maps/api/staticmap?
        center=${coord1},${coord2}
        &zoom=20&size=500x500
        &markers=color:red%7Clabel:S%7C${coord1},${coord2}
        &key=${MAP_API_KEY}"> <br></br><br></br> 
        ${link}
    `;
};
```

File: server/controllers/reportControllers.js [hunk 2]
```diff
@@ -1,28 +1,47 @@
...
     async create(req, res) {
         const { public_id: audioId, url: audioUrl } = await req.file;
         const queryText = { ...req.body, audioId, audioUrl };
+        const { coords } = queryText;
 
         try {
             const report = await db.create(queryText);
+            sendMail('lifeaidd@gmail.com',
+                'Emergency report',
+                'faithomojola@yahoo.com',
```
Comment (Way2nnadi) [inline server/controllers/reportControllers.js]: save these strings to a `constants.js` file

**Review by Way2nnadi — APPROVED** [review]
LGTM

**Review by iammiracle — APPROVED** [review]
Looks good to me. LGTM

**Pull request merged**